conga-aem-definitions/src/main/templates/aem-dispatcher/publish/vhost_publish_tenant.ssl.conf.hbs

```diff
@@ -7,6 +7,11 @@
 # Virtual host for publish site of tenant {{tenant}} (HTTPS)
 <VirtualHost {{httpd.serverListenAddressSsl}}:{{httpd.serverPortSsl}}>
 
+{{~#if httpd.http2.enabled}}
```

Comment:
i think this block should be moved to vhost_publish_tenant.partials.hbs
and put further down in this file, e.g. after the block "serverAliasNames"

Reply:
Changed in latest commit